kolibri/plugins/coach/assets/src/views/common.js

```diff
@@ -66,8 +67,16 @@ export default {
     TimeDuration,
   },
   mixins: [coachStringsMixin],
+  setup() {
+    const { isCoach, isAdmin, isSuperuser, userIsMultiFacilityAdmin } = useUser();
+    return {
+      isCoach,
+      isAdmin,
+      isSuperuser,
+      userIsMultiFacilityAdmin,
```

Comment:
I would prefer to use the composable in the place that this is needed rather than adding it to the mixin (the mixin is bloated and overloaded, so we are trying to phase it out over time).